pkg/provider/containergroup_to_pod.go

```diff
@@ -65,12 +65,13 @@ func (p *ACIProvider) getPodStatusFromContainerGroup(ctx context.Context, cg *az
 			firstContainerStartTime = *containersList[0].Properties.InstanceView.CurrentState.StartTime
 			lastUpdateTime = firstContainerStartTime
 		}
-
+		started := true
 		containerStatus := v1.ContainerStatus{
 			Name:                 *containersList[i].Name,
 			State:                aciContainerStateToContainerState(containersList[i].Properties.InstanceView.CurrentState),
 			LastTerminationState: aciContainerStateToContainerState(containersList[i].Properties.InstanceView.PreviousState),
 			Ready:                getPodPhaseFromACIState(*containersList[i].Properties.InstanceView.CurrentState.State) == v1.PodRunning,
+			Started:              &started,
```

Comment:
How do we know that the container has started in ACI? Why do we assume it will be true?

Reply:
In absence of startup  probe (not supported by ACI), the default value is true. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#containerstatus-v1-core